site/model.md

```diff
@@ -0,0 +1,346 @@
+
+### Declaring Named MBeans to Delete
+
+With WebLogic Deploy Tooling release 1.3.0, you can specify named items in the model to be deleted using the Create Domain, Update Domain, and Deploy Applications Tools.  Named items are those that have multiple instances that are distinguished by user-provided names, such as managed servers, data sources, and security realms.  Items to be deleted are prepended with an exclamation point (!) in the model.
+
+In this example, the managed server ```obsoleteServer``` will be deleted, and ```newServer``` will be created:
+
+```yaml
+    Server:
+        !obsoleteServer:
+        newServer:
+            ListenAddress: 127.0.0.1
+            ListenPort: 9005
+```
+
+If the name of the item requires quotation marks, the exclamation point should be inside the quotation marks:
+
+```yaml
+    Server:
+        '!obsolete-server':
+```
+
+This feature can also remove items that were created by WebLogic Server templates. For example, the base template creates a default security realm called ```myrealm```.  If a user chooses to declare a custom realm, ```myrealm``` is no longer needed.  In this example, ```myrealm``` will be deleted, and the custom realm ```newrealm``` will be created, and declared as the default realm:
+
+```yaml
+    SecurityConfiguration:
+        DefaultRealm: newrealm
+        Realm:
+            !myrealm:
+            newrealm:
+                AuthenticationProvider:
+                ...
+```
+
+This feature does not apply to named security providers within a realm. These items follow a special set of rules that are required to maintain their ordering. See [Modeling Security Providers](site/security_providers.md) for detailed information.
```

Comment:
Link is broken, remove site/ prefix.
Check for others in this file?

Reply:
I will definitely finalize the links after I get the final OK (from Derek) for this re-org, if he likes it. Thanks for pointing it out.